epictrack-web/src/components/workPlan/status/StatusView/RecentStatus.tsx

```diff
@@ -0,0 +1,138 @@
+import React from "react";
+import { Box, Button } from "@mui/material";
+import moment from "moment";
+import { ETCaption1, ETDescription } from "../../../shared";
+import { IconProps } from "../../../icons/type";
+import Icons from "../../../icons";
+import { Palette } from "../../../../styles/theme";
+import { StatusContext } from "../StatusContext";
+import { WorkplanContext } from "../../WorkPlanContext";
+import { useAppSelector } from "../../../../hooks";
+
+const CheckCircleIcon: React.FC<IconProps> = Icons["CheckCircleIcon"];
+const PencilEditIcon: React.FC<IconProps> = Icons["PencilEditIcon"];
+const CloneIcon: React.FC<IconProps> = Icons["CloneIcon"];
+
+const RecentStatus = () => {
+  const { statuses } = React.useContext(WorkplanContext);
+  const { setShowStatusForm, setStatus, setShowApproveStatusDialog } =
+    React.useContext(StatusContext);
+  const { position } = useAppSelector((state) => state.user.userDetail);
+
+  return (
+    <Box
+      sx={{
+        backgroundColor: Palette.neutral.bg.light,
+        padding: "16px 24px",
+        width: "50%",
+        gap: "16px",
+        display: "flex",
+        flexDirection: "column",
+        alignItems: "flex-start",
+      }}
+    >
+      <Box
+        sx={{
+          display: "flex",
+          flexDirection: "row",
+          justifyContent: "space-between",
+          width: "100%",
+        }}
+      >
+        <ETCaption1 bold sx={{ letterSpacing: "0.39px" }}>
+          {moment(statuses[0]?.start_date).format("ll")}
+        </ETCaption1>
+        {statuses[0].approved === false ? (
+          <ETCaption1
+            bold
+            sx={{
+              color: Palette.error.dark,
+              backgroundColor: Palette.error.bg.light,
+              padding: "4px 8px",
+              borderRadius: "4px",
+            }}
+          >
+            Need Approval
+          </ETCaption1>
+        ) : (
```

Comment:
Check out the react-if library, we have it installed and we used it a few times. If you liked it you could use it here, could be more readable

Reply:
willcheck this out